statistic.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# module Statistics
+module Statistics
+  def find_most_popular_element(quantity, item_name)
+    result = @orders
+             .group_by(&item_name)
+             .max_by(quantity) { |_, orders| orders.length }
+             .map(&:first)
+
+    quantity == 1 ? result.first : result
+  end
+
+  def top_reader(quantity: 1)
+    top_reader = find_most_popular_element(quantity, :reader)
+    puts "The top reader is #{top_reader}.\""
```

Comment:
return values from your methods instead of showing text

Reply:
corrected